### PR TITLE
Fix/chat settings modal

### PR DESCRIPTION
**Pull request by iqmulator**

### CODE REVIEW

File: src/dotnet/UI.Blazor/Components/Pic/Pic.razor [hunk 1]
```diff
@@ -20,7 +20,9 @@
 }
 
 <div class="pic @Class @picClass">
-    @if (hasPicture) {
+    @if (IsButton) {
```
Comment (evgeniy-ac) [inline src/dotnet/UI.Blazor/Components/Pic/Pic.razor]: Looks strange. Why do you need it?

File: src/dotnet/UI.Blazor/Components/PicUpload/PicUpload.razor [hunk 2]
```diff
@@ -11,9 +11,10 @@
     var sizeClass = $"pic-size-{(int)Size}";
     var hasButtons = GenerateAvatar.HasDelegate;
     var picUploadCls = hasButtons ? " left-0" : " inset-x-auto";
+    var btnCls = IsButton ? "pic-upload-btn" : "";
```
Comment (evgeniy-ac) [inline src/dotnet/UI.Blazor/Components/PicUpload/PicUpload.razor]: It's an uploader. Why Button?

File: src/dotnet/UI.Blazor/Components/PicUpload/pic-upload.css [hunk 3]
```diff
@@ -48,6 +48,22 @@ body:not(.hoverable) .pic-upload .pic-upload-picker {
     @apply px-2;
 }
 
+.pic-upload-btn {
```
Comment (evgeniy-ac) [inline src/dotnet/UI.Blazor/Components/PicUpload/pic-upload.css:51]: better to put it closer to usage

File: src/dotnet/UI.Blazor/Components/Pic/Pic.razor (hunk 1 above)
Comment (iqmulator) [reply src/dotnet/UI.Blazor/Components/Pic/Pic.razor]: image
This is to turn picuploader into such button.

File: src/dotnet/UI.Blazor/Components/PicUpload/pic-upload.css (hunk 3 above)
Comment (iqmulator) [reply src/dotnet/UI.Blazor/Components/PicUpload/pic-upload.css:51]: This class is applied to picuploader

File: src/dotnet/UI.Blazor/Components/Pic/Pic.razor (hunk 1 above)
Comment (evgeniy-ac) [reply src/dotnet/UI.Blazor/Components/Pic/Pic.razor]: What I want to say in general that you don't have to extend `PicUploader` and `Pic` for a concrete case
IMO it's better to use just a button and `FileUpload` for this case if possible

**Review by alexis-kochetov — APPROVED** [review]

File: src/dotnet/UI.Blazor/Components/Pic/Pic.razor (hunk 1 above)
Comment (iqmulator) [reply src/dotnet/UI.Blazor/Components/Pic/Pic.razor]: Done

**Pull request merged**